backend/routes/billRoutes.js

```diff
@@ -21,4 +21,26 @@ router.post("/", async (req, res) => {
   }
 });
 
+router.get("/", async (req, res) => {
+  try {
+    const bills = await Bill.find();
```

Comment:
this is a an endpoint that returns all bills created by all users? what is the utility of this?

Reply:
I believe i should have put router.get("/:id", async (req, res)
To get a bill and not all bills here

Reply:
yep, that would make more sense.